includes/compat/class-log.php

```diff
@@ -251,11 +253,16 @@ public function file_download_log_get_post_meta( $value, $object_id, $meta_key,
 						$key = 'api_key';
 						break;
 					case 'user':
-						$key = 'user_id';
+						$key = 'customer_id';
+
 						break;
 				}
 
 				$value = $file_download_log->{$key};
+				if ( 'user' === $meta_key ) {
+					$customer = new EDD_Customer( $value );
+					$value    = $customer ? $customer->user_id : 0;
```

Comment:
```suggestion
					$value    = ! empty( $customer->user_id ) ? $customer->user_id : 0;
```

Feels like the better check.